buidler-app/contracts/Payroll.sol

```diff
@@ -57,22 +50,22 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
     string private constant ERROR_LAST_PAYROLL_DATE_TOO_BIG = "PAYROLL_LAST_DATE_TOO_BIG";
     string private constant ERROR_INVALID_REQUESTED_AMOUNT = "PAYROLL_INVALID_REQUESTED_AMT";
 
-    enum PaymentType { Payroll, Reimbursement, Bonus }
-
     struct Employee {
         address accountAddress; // unique, but can be changed over time
         uint256 denominationTokenSalary; // salary per second in denomination Token
+        uint256 denominationTokenAllocation; // allocation paid in denomination token (the rest is in equity)
```

Comment:
Shouldn't  we also keep track the amount paid in equity? so is easier to know how much was paid in equity at certain point?

Reply:
There isn't any need to record this in the contract. However, if it would be useful for the UI somehow then we could discuss it.